docs/features/tuples.md

```diff
@@ -135,7 +135,7 @@ var t = (sum: 0, count: 1);
 System.ValueTuple<int, int> vt = t;  // identity conversion 
 (int moo, int boo) t2 = vt;  		 // identity conversion
 
-t2.moo = 1;
```

Comment:
Why is `moo` invalid?  Its declared as a named item in `(int moo, int boo) t2 = vt;`. There should be no problem assigning to it.